src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -66,11 +66,8 @@
         String lastCommitHash = GitRevList.getCommitHashBeforeDate(
                 config.getRepoRoot(), config.getBranch(), config.getSinceDate());
 
-        if (!lastCommitHash.isEmpty()) {
-            fileInfos = getEditedFileInfos(config, lastCommitHash);
-        } else {
-            fileInfos = getAllFileInfo(config, false);
-        }
+        fileInfos = lastCommitHash.isEmpty() ? getAllFileInfo(config, false)
+                : getEditedFileInfos(config, lastCommitHash);
 
```

Comment:
I think line 194 to line 199 also needs to be changed to use ternary operator right